src/System.Memory/ref/System.Memory.cs

```diff
@@ -9,85 +9,169 @@ namespace System
-        public static System.Span<T> AsSpan<T>(this T[] array, int start, int length) { throw null; }
-        public static System.Span<T> AsSpan<T>(this T[] array, System.Range range) { throw null; }
+        public static System.Span<T> AsSpan<T>(this T[]? array) { throw null; }
+        public static System.Span<T> AsSpan<T>(this T[]? array, System.Index startIndex) { throw null; }
+        public static System.Span<T> AsSpan<T>(this T[]? array, int start) { throw null; }
+        public static System.Span<T> AsSpan<T>(this T[]? array, int start, int length) { throw null; }
+        public static System.Span<T> AsSpan<T>(this T[]? array, System.Range range) { throw null; }
         public static int BinarySearch<T>(this System.ReadOnlySpan<T> span, System.IComparable<T> comparable) { throw null; }
         public static int BinarySearch<T>(this System.Span<T> span, System.IComparable<T> comparable) { throw null; }
-        public static int BinarySearch<T, TComparer>(this System.ReadOnlySpan<T> span, T value, TComparer comparer) where TComparer : System.Collections.Generic.IComparer<T> { throw null; }
+        public static int BinarySearch<T, TComparer>(this System.ReadOnlySpan<T> span, T value, TComparer comparer) where TComparer : System.Collections.Generic.IComparer<T>? { throw null; }
         public static int BinarySearch<T, TComparable>(this System.ReadOnlySpan<T> span, TComparable comparable) where TComparable : System.IComparable<T> { throw null; }
-        public static int BinarySearch<T, TComparer>(this System.Span<T> span, T value, TComparer comparer) where TComparer : System.Collections.Generic.IComparer<T> { throw null; }
+        public static int BinarySearch<T, TComparer>(this System.Span<T> span, T value, TComparer comparer) where TComparer : System.Collections.Generic.IComparer<T>? { throw null; }
         public static int BinarySearch<T, TComparable>(this System.Span<T> span, TComparable comparable) where TComparable : System.IComparable<T> { throw null; }
         public static int CompareTo(this System.ReadOnlySpan<char> span, System.ReadOnlySpan<char> other, System.StringComparison comparisonType) { throw null; }
         public static bool Contains(this System.ReadOnlySpan<char> span, System.ReadOnlySpan<char> value, System.StringComparison comparisonType) { throw null; }
-        public static bool Contains<T>(this System.ReadOnlySpan<T> span, T value) where T : System.IEquatable<T> { throw null; }
-        public static bool Contains<T>(this System.Span<T> span, T value) where T : System.IEquatable<T> { throw null; }
-        public static void CopyTo<T>(this T[] source, System.Memory<T> destination) { }
-        public static void CopyTo<T>(this T[] source, System.Span<T> destination) { }
+        public static bool Contains<T>(this System.ReadOnlySpan<T> span, T value) where T :
+#nullable disable
+            System.IEquatable<T> { throw null; }
+#nullable restore
```

Comment:
In the src I believe we have:
```C#
#nullable disable
    where T : IEquatable<T>
#nullable restore
```
and here we have:
```C#
    where T :
#nullable disable
    IEquatable<T>
#nullable restore
```
Does it matter?

Reply:
Ah true. I don't know if it matters actually. I will update it for consistency in between both, the src and the ref, but we can ask @cston just for the sake of knowing 😄

Reply:
Those two examples should be equivalent since the only type affected by the context is the type reference `IEquatable<T>`.